### PR TITLE
Download JSON library instead of using submodule

### PR DESCRIPTION
**Pull request by alejandro-isaza**

### CODE REVIEW

**Comment by hewigovens** [comment]
can we test it locally instead in the CI? that's may be faster

**Comment by alejandro-isaza** [comment]
I do, but it's not exactly the same. I think it will work now.

**Comment by hewigovens** [comment]
Ok, cool, maybe we need to upgrade Bitrise plan? then downgrade once everything is stable :)

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ macro(find_host_package)
   set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 endmacro(find_host_package)
 
-find_host_package(Boost)
```
Comment (reza1367777) [inline CMakeLists.txt:18]: CMakeLists.txt

Comment (reza1367777) [reply CMakeLists.txt:18]: > CMakeLists.txt

0x69fD9281a920717eE54193a1c130B689Ef341933

**Pull request merged**